ArduCopter/Copter.h

```diff
@@ -631,6 +631,17 @@ class Copter : public AP_Vehicle {
         RELEASE_GRIPPER_ON_THRUST_LOSS = (1<<2),  // 4
         REQUIRE_POSITION_FOR_ARMING =   (1<<3),   // 8
     };
+
+    // type of fast rate attitude controller in operation
+    enum class FastRateType : uint8_t {
+        FAST_RATE_DISABLED            = 0,
+        FAST_RATE_DYNAMIC             = 1,
+        FAST_RATE_FIXED_ARMED         = 2,
+        FAST_RATE_FIXED               = 3,
+    };
+
+    FastRateType get_fast_rate_type() const { return FastRateType(g2.att_enable.get()); }
```

Comment:
What happens if I set a out of range value? Lots of places are checking `!= FAST_RATE_DISABLED` what would a values of -1 result in? Unfortunately we have a presdent that -1 is the same as disabled for lots of params.